btr-web/btr-common-components/composables/useAuth.ts

```diff
@@ -0,0 +1,67 @@
+import { KeycloakConfig } from 'keycloak-js'
+
+/** Manages auth flows */
+export const useAuth = () => {
+  const config = useRuntimeConfig()
+  const keycloak = useBcrosKeycloak()
+  const account = useBcrosAccount()
+  const authenticated = ref(false)
+
+  /** Redirect to bcros login page given the login type. */
+  function goToLogin (idpHint: string) {
```

Comment:
I disagree because these are specific to the auth flow so it makes sense to have them here. Any future options we have for login / logout (other apps would need this) would also go here

Reply:
I really don't like the idea of having a redirect file containing specific flow logic for each redirect

Reply:
Its not flow specific logic, it is certain behavior of redirecting the page to somewhere. I like to keep it in one place, so you can use same behavior from other places. I also see it as clutter in current script.

Reply:
ended up adding these into useNavigation